From 89848b6ee1932d33f505cc9a3ad40ce3e1017087 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Tue, 07 Aug 2018 02:53:53 +0800
Subject: [PATCH] Fix: build failure due to missing include

The macros are defined in sys/sysmacros.h, it may be implicitly
included in previous version of something else.

The following build error is present on Arch without the patch:

```
quick_search/dquicksearch.cpp: In static member function ‘static bool DQuickSearch::isUsbDevice(const QString&)’:
quick_search/dquicksearch.cpp:1034:52: error: ‘major’ was not declared in this scope
         sprintf(link_path, "/sys/dev/block/%d:%d", major(buf.st_rdev), minor(buf.st_rdev));
                                                    ^~~~~
quick_search/dquicksearch.cpp:1034:72: error: ‘minor’ was not declared in this scope
         sprintf(link_path, "/sys/dev/block/%d:%d", major(buf.st_rdev), minor(buf.st_rdev));
                                                                        ^~~~~
```

Change-Id: I09ea552da9aedda63e5128f18a77bf8414053143
---

diff --git a/dde-file-manager-lib/quick_search/dquicksearch.cpp b/dde-file-manager-lib/quick_search/dquicksearch.cpp
index b4f1c70..1a74c25 100644
--- a/dde-file-manager-lib/quick_search/dquicksearch.cpp
+++ b/dde-file-manager-lib/quick_search/dquicksearch.cpp
@@ -24,6 +24,7 @@
 #include <string>
 #include <fstream>
 #include <iomanip>
+#include <sys/sysmacros.h>
 
 #include <zlib.h>
 
